source/documentation/deploying_apps/index.md

```diff
@@ -32,6 +32,19 @@ After deployment, you can increase the running instances to two using:
 
 ### Caveats
 * Your app should not write to local storage. Cloud Foundry local storage is ephemeral and can be deleted at any time.
-* You may need to set environment variables for your app to work. All configuration information should be stored in environment variables, not in the code. 
+* You may need to set environment variables for your app to work. All configuration information should be stored in environment variables, not in the code.
 * Instances will be restarted if they [exceed memory limits](/#quotas).
 * Your application should write all its log messages to `STDOUT`/`STDERR`, rather than a log file.
+
+## Secure and non-secure requests
+
+Requests could be made to the non-secure `http://` protocol due to:
+
+ * misconfiguration of an app that allows non-encrypted traffic through
+ * a service linking to the HTTP version of a page by mistake
+
+In this situation, any requests made to the `http://` protocol will be
+automatically redirected to the base HTTPS version of that URL. Any information
+that is sent as part of that request will be blocked. This ensures that the
```

Comment:
I'm unsure about this because it could feel a bit ambiguous to people that understand HTTPS well:

> Any information that is sent as part of that request will be blocked. 

What do you think about the following?

    The original query path and query parameters will be removed.

Similarly for:

> This ensures that the connection between the client and the server is secure and all information is encrypted.

The original request will have still happened over HTTP. I'm not sure how to represent what @bandesz said about us redirecting to make people aware.

Reply:
Addressed after conversation with Jon and Dan on Hangouts.